views/pages/home.html

```diff
@@ -2,7 +2,8 @@
 <div class="jumbotron">
 	<h1><span class="accented">Unicorn</span> Rampage</h1>
 	<p class="header-caption">The modern day dungeon master helpmate</p>
-	<button class="btn btn-default" type="button">Create</button>
-	<button class="btn btn-default" type="button">Play</button>
+	<a ng-href="/#/create"><button class="btn btn-default" type="button">Create</button></a>
+	<a ng-href="/#/play">
```

Comment:
I've been looking around at solutions to the problem. It seems like a server side solution would be needed. I know in express I could take a url and redirect them so that angular would load andgive them the desired page. Is that what you were thinking as a solution? Or is there a better way?